### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA is not announcing about the context menu is opened in Project Explorer

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1490]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to "Project Explorer" and select required Project  then  Press "Shift+F10" then observe the NVDA behavior

**Expected Result:**
NVDA should announce about the context menu

**Actual Result:**
**NVDA Behavior:**
NVDA is not announcing anything about context menu
**Jaws Behavior:** Jaws is announcing as expected
untitled4  C:\Users\abby\IdeaProjects\untitled4expanded
4 items
menu 
Add Framework Support...
To move through items press up or down arrow.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![menu - jaws](https://user-images.githubusercontent.com/22956130/36135454-4424896c-10b1-11e8-84bc-47e63ede9a91.jpg)
![project-menu-explorer](https://user-images.githubusercontent.com/22956130/36135456-446ba6ee-10b1-11e8-9cdb-1698be921d56.jpg)
![when user hits shift f10 to open context menu nvda is not announcing about context menu](https://user-images.githubusercontent.com/22956130/36135491-74dcea9a-10b1-11e8-8132-a8de69e4c0cc.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749954
